assets/app/scripts/services/data.js

```diff
@@ -550,23 +636,34 @@ angular.module('openshiftConsole')
   // an introspection endpoint that would give us this mapping
   // https://github.com/openshift/origin/issues/230
   var SERVER_TYPE_MAP = {
-    builds:                 API_CFG.openshift,
-    buildConfigs:           API_CFG.openshift,
-    buildConfigHooks:       API_CFG.openshift,
-    deploymentConfigs:      API_CFG.openshift,
-    images:                 API_CFG.openshift,
-    oAuthAccessTokens:      API_CFG.openshift,
-    projects:               API_CFG.openshift,
-    users:                  API_CFG.openshift,
-    routes:                 API_CFG.openshift,
-
-    pods:                   API_CFG.k8s,
-    replicationcontrollers: API_CFG.k8s,
-    services:               API_CFG.k8s,
-    resourcequotas:         API_CFG.k8s,
-    limitranges:            API_CFG.k8s
+    builds:                    API_CFG.openshift,
+    buildConfigs:              API_CFG.openshift,
+    buildConfigHooks:          API_CFG.openshift,
+    deploymentConfigs:         API_CFG.openshift,
+    images:                    API_CFG.openshift,
+    imageRepositories:         API_CFG.openshift,
+    oAuthAccessTokens:         API_CFG.openshift,
+    oAuthAuthorizeTokens:      API_CFG.openshift,
+    oAuthClients:              API_CFG.openshift,
+    oAuthClientAuthorizations: API_CFG.openshift,
+    policies:                  API_CFG.openshift,
+    policyBindings:            API_CFG.openshift,
+    projects:                  API_CFG.openshift,
+    remoteTemplates:           API_CFG.openshift,
+    roles:                     API_CFG.openshift,
+    roleBindings:              API_CFG.openshift,
+    routes:                    API_CFG.openshift,
+    templates:                 API_CFG.openshift,
+    templateConfigs:           API_CFG.openshift,
+    users:                     API_CFG.openshift,
```

Comment:
assuming all the types you added to this list are openshift types and not k8s types?